tests/compatibility-test.py

```diff
@@ -150,18 +153,15 @@ def test_ray_serve(self):
             show_cluster_info(RayFTTestCase.ray_cluster_ns)
             self.fail(f"Fail to execute test_ray_serve_2.py. The exit code is {exit_code}.")
 
+    @unittest.skipIf(
+        ray_version == '2.8.0' or ray_version == 'nightly',
```

Comment:
Why do we need to skip `nightly`?

Reply:
Thanks. That was a mistake. We don’t need to skip nightly anymore.